### ISSUE TITLE
Actual React Component Example

### ISSUE THREAD

**Issue by busypeoples** [jxnblk/mdx-deck#66]
To highlight the power of mdx-deck, it might be good to add an actual interactive React Component to the slides. Just to show that you can run actual code inside these slides. We could even go meta and wrap this component with a provider and update the actual presentation state (f.e. toggle between presenter and non-presenter mode or jump to a specific index etc.)

**Comment by jxnblk** [comment]
Yeah, maybe it's good to start with a basic counter demo – I think we should treat hooking into the app state as private API until things settle a bit; I wouldn't wanna have to introduce breaking changes too soon if people started using that

**Comment by busypeoples** [comment]
Makes sense.
